fvm/storage_capacity_resolver.go

```diff
@@ -0,0 +1,64 @@
+package fvm
+
+import (
+	"fmt"
+
+	"github.com/onflow/cadence/runtime/common"
+	"github.com/onflow/cadence/runtime/interpreter"
+
+	"github.com/onflow/flow-go/fvm/state"
+	"github.com/onflow/flow-go/model/flow"
+)
+
+type StorageCapacityResolver func(state.Ledger, flow.Address) (uint64, error)
+
+func CreateResourceStorageCapacityResolver() StorageCapacityResolver {
+	return func(l state.Ledger, address flow.Address) (uint64, error) {
+		key := fmt.Sprintf("%s\x1F%s", "storage", "storageCapacity") // StorageCapacity resource key. Its the /storage/storageCapacity path
```

Comment:
You can use `interpreter.PathValue` here, and we should export `interpreter.storageKey`